CHANGELOG.md

```diff
@@ -120,3 +120,7 @@ out new cookbooks.
 - REPLAY previously recorded some operations with multiple print statements
 causing REPLAY output from MPI runs to become mixed between multiple ranks.
 REPLAY has been modified to output each operation onto a single line.
+
+- Update calls to allocate_pointer in the FORTRAN bindings to ensure that the
```

Comment:
This should be in the 'Fixed' section at the top of the file (under "Unreleased")

Reply:
@aaroncblack are you able to make this change?

Reply:
?